packages/ui/src/components/Radio/radiogroup.tsx

```diff
@@ -57,16 +60,16 @@ const GroupWithIcon: React.FC<RadioGroupType> = ({ name, options, icon }) => {
 }
 
 const RadioGroup: React.FC<RadioGroupProps> = (props) => {
-    const { groups, icon, ...rest } = props
+    const { groups, children, ...rest } = props
 
     return (
         <DefaultRadio.Group
             {...rest}
             prefixCls={RADIO_GROUP_CLASS_PREFIX}
         >
-            {groups.map(group => (
-                <GroupWithIcon name={group.name} options={group.options} key={group.name} icon={icon} />
-            ))}
+            {groups ? groups.map(group => (
+                <GroupWithIcon name={group.name} options={group.options} key={group.name} />
```

Comment:
GroupWithIcon must be exported as well, since we decided to use children prop

Reply:
I guess it should be <Radio.GroupWithIcon> or smth like that....name is up to you